### DISCUSSION TITLE
Can I select table to backup?

### DISCUSSION THREAD

**Discussion by hxhgxy** [MySqlBackupNET/MySqlBackup.Net#105] [Q&A]
I tried the dll and it works well for the whole database backup.

How can I select some of tables to backup? I don't want to backup all. 
By the way, is it feasible to set password for the sql file?

thanks a lot!
Tiger

**Comment by adriancs2** [comment]
Yes, it can. You may read the documentation below:
https://github.com/MySqlBackupNET/MySqlBackup.Net/wiki/Conditional-Rows-Export-for-Each-Table

**Comment by hxhgxy** [comment]
yes, i got it. thanks very much!
